PyRDF/CallableGenerator.py

```diff
@@ -1,5 +1,3 @@
-from .Operation import Operation
```

Comment:
Watch out, this is already in, please rebase with current master

Reply:
Yup @etejedor . Once you're happy with the other changes, I'll rebase surely 👍